spinalcordtoolbox/vertebrae/core.py

```diff
@@ -288,6 +327,7 @@ def get_z_and_disc_values_from_label(fname_label):
     :param fname_label: image in RPI orientation that contains label
     :return: [z_label, value_label] int list
     """
+    # sct.run('sct_resample -i %s -mm 1x1x1 -x nn -o %s'%(fname_label,fname_label))
```

Comment:
Is this needed?

Reply:
No comes from a previous test will remove.